File: src/net/theharrisoncrafter/Commands/Getenchanting.java
package net.theharrisoncrafter.Commands;

import org.bukkit.ChatColor;
import org.bukkit.command.Command;
import org.bukkit.command.CommandExecutor;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class Getenchanting implements CommandExecutor{
	public boolean onCommand(CommandSender sender, Command cmd, String commandLabel, String[] args){
		
		if(sender instanceof Player && sender.hasPermission("GodCraft.Getenchanging")){
			if(args.length < 0 || args.length > 0){
				return false;
			}
			
			if(((Player) sender).isOnline()){
				((Player) sender).openEnchanting(((Player) sender).getLocation(), true);
				return true;
			}
			return false;
		}
		else{
			sender.sendMessage(ChatColor.RED + "You can not send that command!");
			return true;
		}
	}
}

File: src/main/kotlin/co/orre/godcraft/Commands/GetEnchanting.kt
package co.orre.godcraft.Commands

import co.orre.godcraft.God
import org.bukkit.ChatColor as CC
import org.bukkit.command.Command
import org.bukkit.command.CommandExecutor
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

class GetEnchanting(val plugin: God) : CommandExecutor {
    override fun onCommand(sender: CommandSender, cmd: Command, commandLabel: String, args: Array<String>): Boolean {

        if (sender is Player && sender.hasPermission("godcraft.get_enchanting")) {
            if (args.isNotEmpty()) return false

            if (sender.isOnline) {
                sender.openEnchanting(sender.location, true)
                plugin.logDebug("${sender.name} opened an Enchanting Table window")
                return true
            }
            return false
        }

        sender.sendMessage("${CC.RED}You do not have permissions to do that!")
        return true

    }
}
